src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/block/flower/AumBlock.java

```diff
@@ -0,0 +1,20 @@
+package com.github.minecraftschurlimods.arsmagicalegacy.common.block.flower;
+
+import com.github.minecraftschurlimods.arsmagicalegacy.common.init.AMMobEffects;
+import net.minecraft.world.effect.MobEffect;
+import net.minecraft.world.effect.MobEffects;
+import net.minecraft.world.level.block.Blocks;
+import net.minecraft.world.level.block.FlowerBlock;
+import net.minecraft.world.level.block.state.BlockBehaviour;
+
+public class AumBlock extends FlowerBlock {
```

Comment:
Either add comment referencing MinecraftForge/MinecraftForge#9140 or wait with this PR until it is merged